backend/app/views/spree/admin/taxons/_form.html.erb

```diff
@@ -20,7 +20,24 @@
       </div>
     <% end %>
 
-    <%= render "spree/admin/taxons/attachment_forms/#{f.object.attachment_partial_name}", f: f %>
+    <% if Spree::Config.keep_legacy_backend_support_for_multiple_taxon_attachments %>
```

Comment:
I don't really get why the preference is needed. I think deprecating the partial and those methods in the `_attachment` files is enough.

Reply:
Fixed